src/ccb_neb/acknowledgement.cc

```diff
@@ -71,11 +95,6 @@ acknowledgement::~acknowledgement() {}
  *
  *  @return This object.
  */
-acknowledgement& acknowledgement::operator=(acknowledgement const& other) {
-  io::data::operator=(other);
-  _internal_copy(other);
```

Comment:
Now it is possible to get rid of _internal_copy. the copy constructor does not need this function and can be better.

When it is rewritten, validate it with a unit test.